src/tokens/colors.js

```diff
@@ -0,0 +1,169 @@
+const colors = {
+  azzure: {
+    100: '',
```

Comment:
Let's not keep it empty since it will cause a crash

Reply:
Agreed. Right now we just wanted to see the big picture. We can remove it